### DISCUSSION TITLE
Anyone done a successful AWS ECS deployment?

### DISCUSSION THREAD

**Discussion by data-source** [ludeknovy/jtl-reporter#174] [Q&A]
If so would love to hear some feedback, haven't been able to deploy from minimum viable CF template

**Comment by ludeknovy** [comment]
Hi! I do not personally run it in AWS - but some time ago this tutorial was created: https://jtlreporter.site/docs/guides/aws-ecs-installation (might be already stale though)

**Reply by data-source** [reply]
The very guide I followed... Can't seem to deploy containers, think I'll try from VMs in EC2 instead

**Reply by ludeknovy** [reply]
ahh, ok! If you make it work somehow, please feel free to open MR to https://github.com/ludeknovy/jtl-reporter-docs and share it with the others.

**Reply by data-source** [reply]
I went with an ubuntu image, installed docker, pulled image and docker-compose up -d. Much easier. Done in an hour.

**Reply by ludeknovy** [reply]
Great! Are you willing to create a tutorial for https://github.com/ludeknovy/jtl-reporter-docs ?

**Reply by data-source** [reply]
Just did one https://github.com/ludeknovy/jtl-reporter-docs/commit/4babb2ff71d5839cf61c0bb26c39b1a03d706f8b

**Reply by ludeknovy** [reply]
@data-source Nice, thanks a lot. Could you please open a new pull request so we can merge it and deploy it?

**Reply by data-source** [reply]
Think i created one, not sure it was done correctly though
